implementations/lighthouse/.gitignore

```diff
@@ -1 +1,5 @@
 /vendor
+composer.lock
```

Comment:
Since it may take a while for projects to update I think we should keep the `composer.lock` file committed as otherwise we may start seeing some random failures when one of the deps (or transitive deps) get updated.

Reply:
Sure, let's go with this approach.